Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -297,9 +297,12 @@ void MemoryViewWidget::ToggleBreakpoint()
 
 void MemoryViewWidget::wheelEvent(QWheelEvent* event)
 {
-  int delta = event->delta() > 0 ? -1 : 1;
+  auto delta = -static_cast<int>(std::round((event->angleDelta().y() / 15.)));
```

Comment:
Why 15? Is this because you use 16 as increment/decrement? (which could be `sizeof(u16)` just like the `CodeViewWidget` uses `sizeof(u32)` - which probably makes 15 incorrect up there and it should be 31 if my guess holds...)

Reply:
Because the [Qt docs](http://doc.qt.io/qt-5/qwheelevent.html#angleDelta):


> Most mouse types work in steps of 15 degrees,  in which case the delta value is a multiple of 120; i.e., 120 units * 1/8 = 15 degrees.

I should've noted that somewhere though.